astropy/modeling/core.py

```diff
@@ -2561,6 +2560,7 @@ def __init__(self, op, left, right, name=None, inverse=None):
             self.linear = False
         self.eqcons = []
         self.ineqcons = []
+        self.map_parameters()
```

Comment:
Since this obviates the need for a user to ever all this method, make it private by renaming to _map_parameters?